report_thesis/src/sections/background/linear_and_regularization_models/enet.tex

```diff
@@ -0,0 +1,15 @@
+\subsubsection{Elastic Net Regression (ENet)}
+Despite \gls{lasso} regression's ability to perform feature selection and its effectiveness in settings with multicollinearity, it can struggle when the dataset contains globally highly correlated features.
+\gls{lasso} tends to select only one feature from a group of highly correlated features and ignores the others.
+When multiple correlated features are important for the prediction, this can lead to suboptimal performance.
+
+To address this, \citet{zou_regularization_2005} introduced the \gls{enet} regression, a regression method that combines the $L_1$ and $L_2$ regularization terms from ridge and \gls{lasso} regression.
+The \gls{enet} regression objective function $f_{ENet}(\beta)$ is defined as:
+
+$$
+f_{ENet}(\beta) = \text{RSS} + \lambda_1 L_1 + \lambda_2 L_2,
+$$
+
+where $\text{RSS}$ is the residual sum of squares defined in Section~\ref{sec:ols}, and $\lambda_1$ and $\lambda_2$ are regularization parameters that control the strength of the $L_1$ and $L_2$ penalties, respectively.
```

Comment:
From what I understand, $\lambda_1$ and $\lambda_2$ are actually just a single hyperparameter $r$ called the mix ratio, which is used like so:

![image](https://github.com/chhoumann/thesis-chemcam/assets/29108628/d749ba94-cfb3-4ae7-8a9c-642ebd2c3895)


Does the original paper define it differently?

Reply:
Ah yeah, it does:
![image](https://github.com/chhoumann/thesis-chemcam/assets/29108628/76a28b26-2ae1-4bec-8d77-e31743445109)